scripts/deploy-production.js

```diff
@@ -0,0 +1,92 @@
+const prompts = require("prompts");
+const { exec } = require("child_process");
+
+const localDeployScript =
+  "yarn build:production && env-cmd -e production yarn deploy:do-not-use && env-cmd -e production ./scripts/email.sh";
+
+const main = async () => {
+  // Print all commits between the last tag and the current commit
+  const commitMessages = await getCommitMessages();
+  let shouldDeployLocal = false;
+  if (commitMessages.length > 0) {
+    console.log("Commit messages:");
+    console.log(commitMessages);
+  } else {
+    // If there are no commit messages, ask the user if they want to deploy anyway
+    const response = await prompts({
+      type: "confirm",
+      name: "value",
+      message: "No new commits. Do you want to deploy anyway?",
+      initial: false,
+    });
+    shouldDeployLocal = response.value;
```

Comment:
If the user says no at this point, should we exit the script? Otherwise they get prompted again about deploying:

<img width="438" alt="Screen Shot 2022-11-09 at 11 20 24 AM" src="https://user-images.githubusercontent.com/47064971/200883882-f057ba1e-eea6-41cc-8815-0ab480df71e6.png">